### DISCUSSION TITLE
WSA allegedly shelved due to insufficient Microsoft Store revenue

### DISCUSSION THREAD

**Discussion by infinitepower18** [microsoft/WSA#541] [General]
https://www.windowslatest.com/2024/03/07/windows-11s-android-project-allegedly-shelved-due-to-insufficient-store-revenue/

**Comment by lisyclover** [comment]
They are abandoning their Surface Pro tablet ecosystem, especially Arm's Surface Pro.  Microsoft doesn't have money to fund the good projects, but they can add awfully expensive AI to Paint, Notepad, Store, taskbar, Search, Explorer, .. Oh well.

**Reply by RebelliousX** [reply]
Yeah, a $3 Trillion company that surpasses even Apple, buys 49% shares of OpenAI and can't pay for 3-5 employees $70k-$100k to maintain few projects.

The WSA Kernel has almost 1 million commits (over 980k).  If each commit took literally "1 second" of Microsoft's time, that still will end up as 11 days and 12 hours + few minutes. Time wasted by abandoning this project.

Now there is no easy way to develop Android apps and connect the IDEs (Visual Studio or Android Studio) to WSA via adb once this is abandoned. We have to rely either on the stupid emulator provided by Google or use a real phone. Bummer.

**Comment by Wduran84** [comment]
check out https://windowsxlite.com/downloads/ it has a version of WSA that uses Google Play Store
